pkg/project/resource_project_group_test.go

```diff
@@ -0,0 +1,127 @@
+package project
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/jfrog/terraform-provider-shared/test"
+)
+
+func TestAccProjectGroup(t *testing.T) {
+	projectName := fmt.Sprintf("tftestprojects%s", randSeq(10))
+	projectKey := strings.ToLower(randSeq(6))
+
+	group := "group1"
+
+	resourceName := "project_group." + group
+
+	params := map[string]interface{}{
+		"project_name": projectName,
+		"project_key":  projectKey,
+		"group":        group,
+		"roles":        `["Developer","Project Admin"]`,
+	}
+
+	template := `
+		resource "artifactory_group" "{{ .group }}" {
+			name = "{{ .group }}"
+		}
+
+		resource "project" "{{ .project_name }}" {
+			key = "{{ .project_key }}"
+			display_name = "{{ .project_name }}"
+			description = "test description"
+			admin_privileges {
+				manage_members = true
+				manage_resources = true
+				index_resources = true
+			}
+			max_storage_in_gibibytes = 1
+			block_deployments_on_limit = true
+			email_notification = false
+
+			lifecycle {
+				ignore_changes = ["member"]
+			}
+
+			depends_on = [
+				artifactory_group.{{ .group }}
+			]
+		}
+
+		resource "project_group" "{{ .group }}" {
+			project_key = "{{ .project_key }}"
+			name = "{{ .group }}"
```

Comment:
If you use `name = artifactory_group.{{ .group }}.name` then you won't need the `depends_on` in `project` resource.